<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>PlotToolDialog</class>
 <widget class="QDialog" name="PlotToolDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Make Graphs</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="plot_btn" >
           <property name="text" >
            <string>Make Graph</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="done_btn" >
           <property name="text" >
            <string>Done</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <widget class="QComboBox" name="plot_combox" />
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>Select jig to graph:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="open_trace_file_btn" >
         <property name="text" >
          <string>Open Trace File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="open_gnuplot_btn" >
         <property name="text" >
          <string>Open GNUplot File</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>plot_combox</tabstop>
  <tabstop>plot_btn</tabstop>
  <tabstop>done_btn</tabstop>
  <tabstop>open_trace_file_btn</tabstop>
  <tabstop>open_gnuplot_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>done_btn</sender>
   <signal>clicked()</signal>
   <receiver>PlotToolDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
